windows/hid.c

```diff
@@ -351,14 +349,9 @@ struct hid_device_info HID_API_EXPORT * HID_API_CALL hid_enumerate(unsigned shor
 			if (!res)
 				goto cont;
 
-			if (strcmp(driver_name, "HIDClass") == 0) {
```

Comment:
This section you take out, it's important. It's not exactly like the call to SetupDiGetDeviceRegistryPropertyA() before it, because this one uses SPDRP_DRIVER where the other uses SPDRP_CLASS. This one checks if the driver is bound, which is different.

Please rework so that it checks for both cases.